lib/Twig/Cache/Filesystem.php

```diff
@@ -46,7 +46,7 @@ public function generateKey($name, $className)
      */
     public function load($key)
     {
-        @include_once $key;
+        if (file_exists($key)) include_once $key;
```

Comment:
You can stop now :)

Take any high-loaded website like symfony.com, and try to clear the cache live. Bam!

Anyway, it looks like your error handler is misbehaving here.